src/Utils.js

```diff
@@ -111,7 +111,7 @@ export function lerp(a, b, fract) {
 export const g_randomSeed = 17;
 
 export function randomUnit() {
-  var x = Math.sin(g_randomSeed++) * 10000;
+  let x = Math.sin(this.g_randomSeed++) * 10000;
```

Comment:
You don't have access to `this` here

Reply:
should probably use 'window' it looks like it's supposed to be a global variable

Reply:
and not `const`

Reply:
Did create a const. Pretty sure the default val for g_randomSeed is 17.  Not sure what the return val of randomUnit is supposed to do.